web_tree_background_color/static/src/js/list_view.js

```diff
@@ -0,0 +1,53 @@
+/* Copyright 2018 Leonardo Donelli (donelli@webmonks.it)
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+odoo.define('web_tree_background_color.stuff', function (require) {
+    "use strict";
+
+    var ListView = require('web.ListView');
+    var session = require('web.session');
+
+    ListView.include({
+
+        init: function() {
+            var res = this._super.apply(this, arguments);
+            this.bg_decoration = null;
+            return res;
+        },
+
+        willStart: function() {
+            this.bg_decoration = _.pick(this.fields_view.arch.attrs, function(value, key) {
+                return key.lastIndexOf('decoration-bg-') === 0;
+            });
+            this.bg_decoration = _.mapObject(this.bg_decoration, function(value) {
```

Comment:
you can chain this calls, see example from v11 https://github.com/odoo/odoo/blob/11.0/addons/web/static/src/js/views/list/list_renderer.js#L54